php_mysqlnd_azure.c

```diff
@@ -52,11 +52,31 @@ static ZEND_INI_MH(OnUpdateEnableRedirect)
     return SUCCESS;
 
 }
+
+static ZEND_INI_MH(OnUpdateEnableLogfile) {
+  MYSQLND_AZURE_G(logfilePath) = new_value;
+  return SUCCESS;
+}
+
+static ZEND_INI_MH(OnUpdateEnableLogLevel) {
+  if (STRING_EQUALS(new_value, "3")) {
+    MYSQLND_AZURE_G(logLevel) = 3;
+  } else if (STRING_EQUALS(new_value, "2")) {
+    MYSQLND_AZURE_G(logLevel) = 2;
+  } else if (STRING_EQUALS(new_value, "1")) {
+    MYSQLND_AZURE_G(logLevel) = 1;
+  } else {
+    MYSQLND_AZURE_G(logLevel) = 0;
+  }
+  return SUCCESS;
+}
 /* }}} */
 
 /* {{{ PHP_INI */
 PHP_INI_BEGIN()
 STD_PHP_INI_ENTRY("mysqlnd_azure.enableRedirect", "preferred", PHP_INI_ALL, OnUpdateEnableRedirect, enableRedirect, zend_mysqlnd_azure_globals, mysqlnd_azure_globals)
+STD_PHP_INI_ENTRY("mysqlnd_azure.logfilePath", "mysqlnd_azure_runtime.log", PHP_INI_ALL, OnUpdateEnableLogfile, logfilePath, zend_mysqlnd_azure_globals, mysqlnd_azure_globals)
```

Comment:
It will be great if we can add unit tests to test the option set is correct. You may refer to the ./tests/mysqli_azure_option_test.phpt for reference.

Reply:
And we may also need to print out these two options when call phpinfo(); or php -i like that for enabeRedirect in PHP_MINFO_FUNCTION

Reply:
resolved the `php -i`